java/overrides.js

```diff
@@ -57,5 +57,10 @@
     
     // TODO Failed and needs to be re-checked.
     'appium iOS nav bar check region': {skip: true},
+    'appium android landscape mode check window andorid 7': {skip: true},
+    'appium android landscape mode check region android 7': {skip: true},
+    'appium android landscape mode check window andorid 10': {skip: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9250217)
```suggestion
    "appium android landscape mode check window andorid 10": {skip: true},
```